File: CS/NavigationPanelOptions/Form1.cs
using System;
using System.Windows.Forms;

namespace NavigationPanelOptions {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        #region #NavigationPanelOptions
        private void Form1_Load(object sender, EventArgs e) {
            mapControl1.NavigationPanelOptions.XCoordinatePattern = "{CP}{D}.{M}.{S:4}";
            mapControl1.NavigationPanelOptions.YCoordinatePattern = "{CP}{D}.{M}.{S:4}";
            mapControl1.NavigationPanelOptions.ShowKilometersScale = false;
            mapControl1.NavigationPanelOptions.ShowMilesScale = false;
            mapControl1.NavigationPanelOptions.ShowScrollButtons = false;
        }
        #endregion #NavigationPanelOptions
    }
}

File: VB/NavigationPanelOptions/Form1.vb
Imports System
Imports System.Windows.Forms

Namespace NavigationPanelOptions
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        #Region "#NavigationPanelOptions"
        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            mapControl1.NavigationPanelOptions.XCoordinatePattern = "{CP}{D}.{M}.{S:4}"
            mapControl1.NavigationPanelOptions.YCoordinatePattern = "{CP}{D}.{M}.{S:4}"
            mapControl1.NavigationPanelOptions.ShowKilometersScale = False
            mapControl1.NavigationPanelOptions.ShowMilesScale = False
            mapControl1.NavigationPanelOptions.ShowScrollButtons = False
        End Sub
        #End Region ' #NavigationPanelOptions
    End Class
End Namespace
